nana/assistant/eval.py

```diff
@@ -3,30 +3,65 @@
 import os
 import re
 import subprocess
+from io import StringIO
 
-from pyrogram import Filters
+from pyrogram import filters
 
 from nana import Owner, logging, setbot
 from nana.modules.devs import aexec
 
 
-@setbot.on_message(Filters.user(Owner) & Filters.command(["py"]))
-async def executor(client, message):
-    if len(message.text.split()) == 1:
-        await message.reply("Usage: `/py await message.reply('edited!')`")
+@setbot.on_message(filters.user(Owner) & filters.command(["eval"]))
+async def eval(client, message):
```

Comment:
Codacy found an issue: [Redefining built-in 'eval'](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)